<?xml version="1.0"?>
<launch>

	<param name="use_sim_time" value="true"/>
	<node name="ddpg" pkg="grasp_learning" type="ddpg.py" output="screen">
	 	<param name="batch_size" value="2" />
 		<param name="num_inputs" value="2" />
		<param name="num_outputs" value="2" />
		<param name="relative_path" value="$(find grasp_learning)/stored_data/tested_data/" />
 	</node>
</launch>
